Add Solosu and Yoso to wiki (#20)
Rurusetto/rurusetto: content/posts/bosu/index.md

  - The song speed will be getting faster and faster
  - Click the icon again for `Wind Down Mod` : Slooow doooown...
    - Slowly decreases speed from 1.00x to 0.75x throughout the map and lowers the pitch alongside the speed.

>   - You can make a customisation on *Initial rate*, *Final rate*, and *Adjust pitch*

Reviewer: ```suggestion
  - *Initial rate*, *Final rate*, and *Adjust pitch* are customizable parameters.
```
